vcf/queryInfo_test.go

```diff
@@ -200,10 +200,13 @@ func TestQueryInt(t *testing.T) {
 	v := ParseFormat(data[0], header)
 	v = ParseInfo(v, header)
 
-	if !equalInt(QueryInt(v, header.Info["InfoA"].Key), expectedInfo["InfoA"]) {
+	val, found := QueryInt(v, header.Info["InfoA"].Key)
+	if !found || !equalInt(val, expectedInfo["InfoA"]) {
 		t.Errorf("troble querying read info")
```

Comment:
Are all these error messages supposed to say "trouble querying..."?

Reply:
nice catch. The dangers of copypasta